padertorch/contrib/examples/tasnet/tasnet.py

```diff
@@ -226,3 +226,54 @@ def review(self, inputs, outputs):
 
     def flatten_parameters(self) -> None:
         self.dprnn.flatten_parameters()
+
+
+class ModularTasNet(TasNet):
```

Comment:
Is the opposite inheritance not better? `TasNet` initialize encoder separator and decoder and forwards them to `ModularTasNet`.

For now I like to keep `TasNet` to make no breaking change. But in the long term, we may deprecate it.

Reply:
I agree with this

Reply:
yes, the idea is to rework this after the workshop

Reply:
Do you mean deprecation or changing the inheritance order?

Reply:
deprecation, I can change the inheritance now

Reply:
ok, thanks